package.json

```diff
@@ -35,17 +35,20 @@
   "dependencies": {
     "ember-cli-babel": "6.6.0",
     "ember-promise-tools": "1.0.0",
-    "ember-runtime-enumerable-includes-polyfill": "2.0.0",
-    "ember-string-ishtmlsafe-polyfill": "^1.1.0"
```

Comment:
I removed because I got this message running `ember serve` locally. 

```
 ember serve
ember-string-ishtmlsafe-polyfill is not required for Ember 2.8.0 and later, please remove from your `package.json`.
```
Seems like its necessary for backwards compatibility so I'll add it back in.